### PR TITLE
updates for clarity on daemonset alternatives

### PR DESCRIPTION
**Pull request by tkellen**
I'd be happy to update the prose here with a bit of guidance!

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6240)

### CODE REVIEW

**Comment by steveperry-53** [comment]
@tkellen I'll be glad to help with this. Can you clarify what guidance you'd like me to provide? Thanks.

**Comment by tkellen** [comment]
Hi @steveperry-53, thanks for chiming in! Let me see if I can put a fine point on what I was confused about...

[This section](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/#updating-a-daemonset) reads:
> In Kubernetes version 1.6 and later, you can perform a rolling update on a DaemonSet.
> Future releases of Kubernetes will support controlled updating of nodes.

[This section](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/#init-scripts) (where I left my note) reads:
> Future versions of Kubernetes will likely support integration between DaemonSet-created Pods and node upgrade workflows.

It seems to me that the first section is a confirmation of the second, and that the second should be removed, or expanded somehow.

What do you think?

**Comment by tengqm** [comment]
I'd like to vote for removing both. The documentation should be only about what things are today, and even better, what things were yesterday. Any prediction about the future might risk being false hope.

**Comment by tkellen** [comment]
I've taken your vote and implemented it @tengqm!

**Comment by tengqm** [comment]
rebase needed, otherwise lgtm.

**Comment by tkellen** [comment]
Whoops. Updated!

File: docs/concepts/workloads/controllers/daemonset.md [hunk 1]
```diff
@@ -158,10 +158,6 @@ The new DaemonSet with the different template will recognize all the existing Po
 matching labels.  It will not modify or delete them despite a mismatch in the Pod template.
 You will need to force new Pod creation by deleting the Pod or deleting the node.
 
-In Kubernetes version 1.6 and later, you can [perform a rolling update](/docs/tasks/manage-daemon/update-daemon-set/) on a DaemonSet.
```
Comment (tengqm) [inline docs/concepts/workloads/controllers/daemonset.md]: Should we keep this line?

Comment (tkellen) [reply docs/concepts/workloads/controllers/daemonset.md]: Seems reasonable, yes. I've just restored it.

**Pull request merged**